crates/containerd-shimkit/src/vendor/containerd_shim/logger.rs

```diff
@@ -87,6 +87,14 @@ impl SimpleWriteVisitor {
     }
 }
 
+pub(super) static KEY_VALUES: Mutex<String> = Mutex::new(String::new());
```

Comment:
question: why do you need a Mutex to protect the KEY_VALUES?

Reply:
Yes, otherwise I would need to use unsafe